workshops/boost-research-reproducibility-binder/reports/workshop_report_20200131.md

```diff
@@ -0,0 +1,70 @@
+# Feedback report on The Turing Way 'Boost your Research Reproducibility with Binder' workshop | January 2020
+
+Kirstie Whitaker - representing _The Turing Way_ team - held a 'Boost your Research Reproducibility with Binder' workshop at the [British Antarctic Survey](https://www.bas.ac.uk) in Cambridge on 31 January 2020.
+
+The half day workshop was organised by [Dr Anita Faul](https://www.bas.ac.uk/profile/anfaul/) - a data scientist and machine learning expert at BAS.
+It was a closed event for BAS team members, but Kirstie's talk was recorded and is available for anyone to view at *insert youtube link*.
+The workshop went smoothly all thanks to Anita's hard work - thank you!
+
+The agenda for this [British Antarctic Survey: 31 January 2020](../agenda.md#british-antarctic-survey-31-january-2020) workshop includes with links to PDFs of the presentations and markdown files for the code-along exercises.
+
+## Instructor and team reflections
+
+Kirstie re-did the slides for the introduction and "Why you need a reproducible computing environment and how Binder can help" presentations ([IntroBoostResReproBinder_20200131.pdf](../workshop-presentations/IntroBoostResReproBinder_20200131.pdf) and [ReproducibleComputationalEnvironment_20200131.pdf](../workshop-presentations/ReproducibleComputationalEnvironment_20200131.pdf)) before this workshop so they're now more interoperable with other talks about the project.
+They use the Turing Institute template and include a doi and twitter hashtag.
+
+There was a major snafu with the second of the [paired examples](paired_examples.md) exercise.
+It turns out that the [CompEnv-Ex2](https://github.com/alan-turing-institute/CompEnv-Ex2) one relies on a very old (and no-longer supported) version of matplotlib and (probably) as a result of python 2 being retired on [1 January 2020](https://pythonclock.org/) the binder with that old version didn't build.
+
+* Fortunately Kirstie realised this before the workshop so was prepared to ask participants to skip that example.
+* Huge thank you to Turing Way team member and Binder operator Sarah Gibson who tried to fix the problem during the workshop (after a last minute request on Gitter).
+  And thank you to the workshop participants for their understanding of the problem.
+* Fortunately for future workshops, Sarah [opened an issue](https://github.com/alan-turing-institute/CompEnv-Ex2/issues/1) and [fixed the problem](https://github.com/alan-turing-institute/CompEnv-Ex2/pull/2)! 🙌 ✨ 👾
+* The lesson to learn is to double check all the exercises well in advance of the workshop.
+  Kirstie ended up missing out on some very interesting discussions because she was trying to multitask and fix the error at the same time.
+
+Taking a team photo was really wonderful!
+Let's try to make sure that's in all future workshops.
+
+![](https://raw.githubusercontent.com/scott-hosking/my-first-binder/master/BoostReproducibilityBinder_BAS_2020-01-31.jpg)
+
+*Add Anita's comments here.*
```

Comment:
very happy with the workshop and impressed by the efforts trying to fix the example which wasn't working in real time. Kirstie was very engaged and happy to answer all questions. There were also interesting discussions over lunch and at coffee. In hindsight, it would have been useful to highlight the software carpentry lessons (https://software-carpentry.org/lessons/) for people to work through before the workshop. or run one of those workshops before this one.

Reply:
Added (in a new PR because this was accidentally merged!) Thanks @ACFaul